src/Package/Impl/ProjectSystem/Commands/CopyItemPathCommand.cs

```diff
@@ -46,5 +47,7 @@ internal sealed class CopyItemPathCommand : IAsyncCommandGroupHandler {
 
             return true;
         }
+
+        private IMainThread MainThread => _interactiveWorkflowProvider.GetOrCreate().Shell.MainThread();
```

Comment:
Let's import `ICoreShell` here explicitly - I don't want create `RInteractiveWorkflow` instance for an assert